mmv1/third_party/terraform/utils/provider_test.go.erb

```diff
@@ -231,15 +235,18 @@ func getCachedConfig(ctx context.Context, d *schema.ResourceData, configureFunc
 		return false
 	})
 	config.client.Transport = rec
-	mutex.Lock()
+	configs_lock.Lock()
```

Comment:
I believe camelCase is the style guideline for golang

Reply:
Yeah. I'll do that!